### PR TITLE
team: add new reviewers of eg

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

team: add new reviewers of eg

**What this PR does / why we need it**:

Add new reviewers of eg.

**Which issue(s) this PR fixes**:

Fixes https://github.com/envoyproxy/gateway/issues/2020

### CODE REVIEW

**Comment by Xunzhuo** [comment]
Hi forks, @alyssawilk @phlax ! Would you like to help invite to envoyproxy org and @envoyproxy/gateway-reviewers team?

+ @tanujd11 
+ @tmsnan 
+ @cnvergence

File: OWNERS [hunk 1]
```diff
@@ -22,3 +22,7 @@ reviewers:
 - LanceEa
 - qicz
```
Comment (zirain) [inline OWNERS:23]: remove qicz from here?

**Comment by alyssawilk** [comment]
invites sent!

File: OWNERS (hunk 1 above)
Comment (Xunzhuo) [reply OWNERS]: Done.

**Review by arkodg — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**